changelog.txt

```diff
@@ -1,3 +1,6 @@
+Version: 2.0.1
+Game Versions: v1.2.x
+* Fix : Factionless npcs dialogues should no longer cause crashes. Improves compatibility with non-Custom Spawns mods.
```

Comment:
New formulation. Opted for "should" over "will" because I don't like the absolute formulation; I'm not *that* certain of the fix.